src/getDataTransferItems.spec.js

```diff
@@ -19,13 +19,16 @@ const files = [
 ]
 
 describe('getDataTransferFiles', () => {
-  it('should return an array', () => {
+  it('should return an promise with an array', done => {
```

Comment:
Please consider using http://facebook.github.io/jest/docs/en/asynchronous.html#promises for tests here.